lib/rules/declaration-property-value-no-unknown/__tests__/index.mjs

```diff
@@ -138,6 +207,36 @@ testRule({
 			endLine: 1,
 			endColumn: 30,
 		},
+		{
+			code: `
+				a { --foo: 10px; }
+				@property --foo {
+					syntax: "<color>";
+					inherits: false;
+					initial-value: #c0ffee;
+				}
+			`,
+			message: messages.rejected('--foo', '10px'),
+			line: 2,
+			column: 16,
+			endLine: 2,
+			endColumn: 20,
```

Comment:
[suggest] This isn't required, but I suggest using `stripIndent` to easily check `column` values. The helper removes TAB characters in a template string literal. For example:

https://github.com/stylelint/stylelint/blob/c1aefb4d5c3c15d035a658a8e6f53be2fd5de283/lib/rules/color-function-notation/__tests__/index.mjs#L129-L136

Reply:
Nice, thank you 🙇